ml_server/.pre-commit-config.yaml

```diff
@@ -0,0 +1,27 @@
+fail_fast: true
+repos:
+  - repo: https://github.com/pycqa/isort
+    rev: 5.13.2
+    hooks:
+    - id: isort
+      args:
+        - --profile=black
+        - --filter-files
+        - --line-length=79
+  - repo: https://github.com/psf/black
+    rev: 22.3.0
+    hooks:
+      - id: black
+        args:
+          - --line-length=79
+  - repo: https://github.com/pycqa/flake8
+    rev: 7.1.2
+    hooks:
+      - id: flake8
+        args:
+          - --extend-ignore=E203
```

Comment:
let's add these checks to CI as well

Reply:
Additionally, no need to specify args for linters here, they should have been parsed from pyproject.toml

Reply:
I used CI config from other services, Unfortunately I don't have time for setting up and testing new CI pipelines for a service and syncing it with pyproject.toml. All other services are having this logic, I took it from there.